src/storage/migration/migration_structures.rs

```diff
@@ -284,3 +285,10 @@ impl From<TlsBeforeV21> for TlsFromV21 {
         }
     }
 }
+
+#[derive(Deserialize, Serialize)]
+pub struct OplogBeforeV22 {
```

Comment:
I think it is before V23.

Reply:
I'm not sure if Giganto version will be version 23, so I made it 22. 
I'll change it to 23.

Reply:
Thank you for the change.

Just a quick note: if a migration occurs, the major version goes up. It seems you are aware of this since you've updated `COMPATIBLE_VERSION_REQ` to `>=0.23.0,<0.24.0`. However, I think it's worth explicitly mentioning this  to avoid any potential confusion.